src/ansys/sherlock/core/types/layer_types.py

```diff
@@ -0,0 +1,71 @@
+# © 2023 ANSYS, Inc. All rights reserved.
+
+"""Module containing types for the Layer Service."""
+
+
+class PolygonalShape:
+    """Contains the properties for a polygonal shape."""
+
+    def __init__(self, points, rotation):
+        """Initialize the shape properties."""
+        self.points = points
+        """points (length 2 tuples) : list"""
```

Comment:
This should specify X and Y values so the order is defined since named attributes aren't being used.